webrender/src/frame_builder.rs

```diff
@@ -882,12 +882,17 @@ impl FrameBuilder {
                     BoxShadowClipMode::Inset => 1.0,
                 };
 
-                // If we have a border radius, we'll need to apply
-                // a clip-out mask.
+                // Outset box shadows with border radius
+                // need a clip out of the center box.
+                let extra_clip_mode = match clip_mode {
```

Comment:
Ah, you're right - and we only apply this is radius > 0.0, so it should be fine. Thanks!